lib/domain.js

```diff
@@ -123,10 +107,10 @@ function topLevelDomainCallback(cb, ...args) {
   return ret;
 }
 
-// It's possible to enter one domain while already inside
-// another one. The stack is each entered domain.
-const stack = [];
-exports._stack = stack;
+
+// It's possible to enter one domain while already inside another one. The stack
+// is each entered domain.
+const stack = exports._stack = process._domainsStack;
```

Comment:
can you revert the change to this line?

Reply:
What do you mean? How would `stack` and `exports._stack` be initialized?

Reply:
The general goal is to avoid doing `x = y = z`

this would work:
```js
const stack = process._domainStack;
exports._stack = stack; // no change to this line
```

or better yet, remove the `const stack =` entirely and just refer to `process._domainStack`.

Reply:
Sounds good, will do!